options/locale/locale_en-US.ini

```diff
@@ -1080,8 +1080,8 @@ dashboard.delete_missing_repos = Delete all repository records which have lost t
 dashboard.delete_missing_repos_success = All repository records which have lost their corresponding Git files have been deleted.
 dashboard.git_gc_repos = Execute garbage collection on all repositories
 dashboard.git_gc_repos_success = All repositories have finished executing garbage collection.
-dashboard.resync_all_sshkeys = Rewrite '.ssh/authorized_keys' file (for Gitea SSH keys)
-dashboard.resync_all_sshkeys_success = All public keys have been rewritten.
+dashboard.resync_all_sshkeys = Rewrite '.ssh/authorized_keys' file (for Gitea SSH keys). Do not use this if you are using the built-in SSH server.
```

Comment:
This should be `Disabled if you are using the built-in SSH server.` no? As in it won't do anything...